foreign_born.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
Have you been able to refactor your code and open a new pull request?

Reply:
Hello Joseph, I am working on the changes. will upload the new ones soon.